### PR TITLE
Update installation instructions

### PR DESCRIPTION
**Pull request by lexidor**
Addresses #678

### CODE REVIEW

File: guides/hhvm/01-installation/02-linux.md [hunk 1]
```diff
@@ -53,17 +41,31 @@ apt-get update
...
+## Obtaining A Minor Release
+
+It is generally recommended to follow the newest version possible, provided your codebase is compatible with that version. You can fetch all supported versions (except for HHVM 3.30) by adding that version to `/etc/apt/sources.list`. The syntax you need is:
+
+`deb https://dl.hhvm.com/<%operating system%> <%operating system version%>-<%major%>.<%minor%>`
+
+So in order to get HHVM 4.8 on ubuntu bionic (18.04) you would use
+`deb https://dl.hhvm.com/ubuntu bionic-4.8`
+
+You will automatically receive patches such as HHVM 4.8.1, but your won't upgrade to HHVM 4.9 and up.
```
Comment (fredemmott) [inline guides/hhvm/01-installation/02-linux.md]: s/your/you/

File: guides/hhvm/01-installation/02-linux.md [hunk 2]
```diff
@@ -53,17 +41,31 @@ apt-get update
 apt-get install hhvm
 ```
 
-## Obtaining LTS Releases
+## Obtaining A Minor Release
```
Comment (fredemmott) [inline guides/hhvm/01-installation/02-linux.md]: Maybe 'Specific Minor' or just 'Specific' ?

File: guides/hhvm/01-installation/02-linux.md [hunk 3]
```diff
@@ -53,17 +41,31 @@ apt-get update
...
+`deb https://dl.hhvm.com/<%operating system%> <%operating system version%>-<%major%>.<%minor%>`
+
+So in order to get HHVM 4.8 on ubuntu bionic (18.04) you would use
+`deb https://dl.hhvm.com/ubuntu bionic-4.8`
+
+You will automatically receive patches such as HHVM 4.8.1, but your won't upgrade to HHVM 4.9 and up.
+
+In order to get HHVM 3.30 use:
+`deb https://dl.hhvm.com/ubuntu bionic-lts-3.30 main`
+This is how lts release were previously released.
```
Comment (fredemmott) [inline guides/hhvm/01-installation/02-linux.md]: LTS

**Comment by lexidor** [comment]
> Thanks! As well as the in-line comments, could you adjust the new instructions to use `add-apt-repository` instead please?
> 
> e.g.:
> 
> ```
> add-apt-repository "deb http://dl.hhvm.com/$(lsb_release -si | tr A-Z a-z) $(lsb_release -sc)-4.8 main"
> ```

I think I understood what you meant, but I might not have. If 37d8d52 is not what you meant, I am sorry.

**Comment by fredemmott** [comment]
What's here was fine, but that line should have worked verbatim - `lsb_release -si` should output 'Debian' or 'Ubuntu' on supported systems, and `lsb_release -sc` should output codenames like 'bionic'

**Pull request merged**